src/functional_test_runner/lib/config/schema.js

```diff
@@ -79,7 +79,7 @@ export const schema = Joi.object().keys({
 
   timeouts: Joi.object().keys({
     find: Joi.number().default(10000),
-    try: Joi.number().default(40000),
+    try: Joi.number().default(120000),
```

Comment:
What's the purpose of this change?

Reply:
Well, this does fix the flakiness for me.  I suspect it's because some of the rendering tests are VERY VERY slow.  Maybe 2 minutes is more than neccessary, but, check out these tests I am running to track performance: The one takes 43 seconds, so it could really take over 40 seconds for a visualization to render (or, 30 visualizations on a dashboard to all finish rendering)

<img width="1252" alt="screen shot 2019-02-07 at 8 59 17 pm" src="https://user-images.githubusercontent.com/16563603/52454828-629bcd80-2b1b-11e9-98b4-d0742214d216.png">

Reply:
eh, this might not even fix it, I hit:
```
final test name is  dashboard embeddable rendering adding visualizations
└-> "before each" hook: global before each
└- ✖ fail: "dashboard app using current data dashboard embeddable rendering adding visualizations"
│      Error: Timeout of 180000ms exceeded. For async tests and hooks, ensure "done()" is called; if returning a Promise, ensure it resolves.
│
│
```

now.  but when it passes it takes around ~40s so that should be sufficient.

Reply:
Not sure if that one failure was a anomoly, but here is why retry timeout needs to be longer:
```
      await retry.try(async () => {
        const completedElements = await find.allByCssSelector(RENDER_COMPLETE_SELECTOR);
        if (completedElements.length < count) {
          throw new Error(`${completedElements.length} elements completed rendering, waiting on a total of ${count}`);
        }

        const stillLoadingElements = await find.allByCssSelector(DATA_LOADING_SELECTOR, 1000);
        if (stillLoadingElements.length > 0) {
          throw new Error(`${stillLoadingElements.length} elements still loading contents`);
        }
      });
```

esp when we introduced the change that caused slowness in visualize rendering, this could take a long time to finish rendering everything on the page.